site/public/js/calendar.js

```diff
@@ -772,3 +772,21 @@ function colorLegend() {
     });
 }
 
+//args contains course id and course name. We want to use display_name is possible
```

Comment:
```suggestion
// Modifies cookies so the correct filtering of courses on the calendar is chosen.
// param string course_val Id of the course
// param string display_name display name of the course
```

This is what documenting arguments should usually look like. I'm following my suggestion in this comment of spliting the arguments up.